src/validation/forks.h

```diff
@@ -21,6 +21,8 @@ class CBlockIndex;
 class CScript;
 class CTxMemPoolEntry;
 
+extern CChain chainActive;
```

Comment:
This looks out of place. If this is due to having issues including `main.h`, then there is probably a deeper code coupling problem that needs to be solved.

Reply:
This will be solved by passing param and current tip to the `AreWeOnBCHChain()` as you sugegest here https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/1644#discussion_r272368068

Reply:
done, see fa23e17